.NET/Microsoft.Recognizers.Definitions.Common/Swedish/QuotedTextDefinitions.cs

```diff
@@ -25,7 +25,7 @@ public static class QuotedTextDefinitions
       public const string QuotedTextRegex1 = @"(“([^“”]+)”)";
       public const string QuotedTextRegex2 = @"(‘([^‘’]+)’)";
       public const string QuotedTextRegex3 = @"(""([^""]+)"")";
-      public const string QuotedTextRegex4 = @"('([^']+)')";
+      public const string QuotedTextRegex4 = @"(\\'([^\']+)\\')";
```

Comment:
Can you revert this change? This escaping is invalid and is not automatically generated in the CI build.

Reply:
I just went ahead and reverted and re-triggered the PR build.